tests/providers/gcp/gcp_test.py

```diff
@@ -708,11 +741,35 @@ def testGetObjectMetadata(self, mock_gcs_api):
     """Test GCS object Get operation."""
     api_get_object = mock_gcs_api.return_value.objects.return_value.get
     api_get_object.return_value.execute.return_value = MOCK_GCS_OBJECT_METADATA
-    get_results = FAKE_GCS.GetObjectMetadata('gs://Fake_Path')
+    get_results = FAKE_GCS.GetObjectMetadata('gs://fake-bucket/foo/fake.img')
     self.assertEqual(MOCK_GCS_OBJECT_METADATA, get_results)
     self.assertEqual('5555555555', get_results['size'])
     self.assertEqual('MzFiYWIzY2M0MTJjNGMzNjUyZDMyNWFkYWMwODA5YTEgIGNvdW50MQo=', get_results['md5Hash'])
 
+  @typing.no_type_check
+  @mock.patch('libcloudforensics.providers.gcp.internal.storage.GoogleCloudStorage.GcsApi')
+  def testListBucketObjects(self, mock_gcs_api):
+    """Test GCS object List operation."""
+    api_list_object = mock_gcs_api.return_value.objects.return_value.list
+    api_list_object.return_value.execute.return_value = MOCK_GCS_BUCKET_OBJECTS
+    list_results = FAKE_GCS.ListBucketObjects('gs://fake-bucket')
+    self.assertEqual(len(list_results), 1)
```

Comment:
Please format assertions as `self.assertEqual(expected, actual)`: `self.assertEqual(1, len(list_results))` so that it is consistent with the other tests.

Same comment for the assertions down below in `testGetBucketACLs`.